astropy/cosmology/_src/flrw/base.py

```diff
@@ -359,7 +361,7 @@ def Onu0(self) -> float:
     # ---------------------------------------------------------------
 
     @deprecated_keywords("z", since="7.0")
-    def Otot(self, z):
+    def Otot(self, z: u.Quantity | ArrayLike) -> FArray | float:
```

Comment:
`float` or `np.floating` or both?

Reply:
I am going to trust that you have a plan.

Reply:
It's to go all in on the Array API.
Flexible inputs. Always Array API outputs.